stringfuzz/generators/regex.py

```diff
@@ -164,20 +217,35 @@ def make_regex(
     if max_var_length is not None and max_var_length < 0:
         raise ValueError('max variable length must not be less than 0')
 
+    if len(operators) < 1 or any(map(lambda x: x not in OPERATOR_LIST, operators)):
```

Comment:
Nitpick: `any` and `map` are still a bit difficult to read, and I was thinking of something along the lines of:
```python
invalid_operators = set(operators).difference(set(OPERATOR_LIST))
if len(operators) < 1 or len(invalid_operators) > 1:
    ...
```
However both are fine.

Reply:
I personally prefer the functional version for readability, so I'll leave it.